pages/about/.backups/index.html~

```diff
@@ -0,0 +1,15 @@
+<!-- See structure.html for the template names used here and their position in the layout. -->
```

Comment:
You probably did not intend to add this one here.

Reply:
Oops, sorry. Removed in a (squashed) commit.